swf/models/workflow.py

```diff
@@ -246,7 +246,7 @@ def start_execution(self, workflow_id=None, task_list=None,
         :type   input: dict
 
         :param  tag_list: Tags associated with the workflow execution
-        :type   tag_list: String
+        :type   tag_list: String or list of strings
```

Comment:
The number of tags must be less than or equal to 5.
Might be important to mention in doc-string or better to force on the code site.

Otherwise you will get the following error:
```
boto.exception.SWFResponseError: SWFResponseError: 400 Bad Request
{u'message': u"1 validation error detected: Value '[t1, t2, t3, t4, t5, t6]' at 'tagList' failed to satisfy constraint: Member must have length less than or equal to 5", u'__type': u'com.amazon.coral.validate#ValidationException'}
```